lib/lnc.ts

```diff
@@ -71,6 +71,10 @@ export default class LNC {
         return globalThis[this._namespace] as WasmGlobal;
     }
 
+    private set wasm(value: any) {
+        globalThis[this._namespace] = value;
+    }
+
```

Comment:
This is just a shortcut to avoid writing `globalThis[this._namespace] = {}` in the `run` function, which makes it more understandable IMO. I originally wrote the code as: 
```js
if (typeof globalThis[this._namespace] !== 'object') {
  globalThis[this._namespace] = {};
}

if (!this.wasm.onLocalPrivCreate) {
  this.wasm.onLocalPrivCreate = (keyHex: string) => {
    log.debug('local private key created: ' + keyHex);
    this.credentials.localKey = keyHex;
  };
}
````

I realized that it wouldn't be obvious to the reader that `globalThis[this._namespace]` and `this.wasm` referred to the same thing. So i just added the setter to make that more clear.